test_field_agents_architecture.py

```
@@ -0,0 +1,226 @@
+    for i, case in enumerate(test_cases, 1):
+        print(f"\n--- テスト {i}: {case['description']} ---")
+        print(f"入力: {case['query']}")
+        
+        try:
+            result = await agent.process_query(case['query'])
+            print(f"成功: {result['success']}")
+            print(f"タイプ: {result.get('query_type', 'unknown')}")
+            print(f"応答: {result['response'][:100]}...")
+            
+            if case['expected'] in ['registration_redirect', 'out_of_scope']:
+                assert not result['success'], f"期待: 失敗, 実際: 成功"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)